### PR TITLE
add cypress to project with demo test

### PR DESCRIPTION
**Pull request by Rayman-bat9**

### CODE REVIEW

**Comment by VadimNastoyashchy** [comment]
Description of the PR should be populated

**Comment by VadimNastoyashchy** [comment]
Reviewers should be selected

**Comment by VadimNastoyashchy** [comment]
Assignees should be selected (who was created this pr)

File: package-lock.json
Comment (VadimNastoyashchy) [inline package-lock.json:1]: package-lock.json -  file should also be added to the git ignore file. Will be automatically created by CI/CD

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "scripts": {
+    "test": "cypress open",
+    "test:run": "cypress run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/VadimNastoyashchy/cypress-aqa-cource.git"
+  },
+  "keywords": [],
+  "author": "",
```
Comment (VadimNastoyashchy) [inline package.json]: might me populated

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "cypress open",
+    "test:run": "cypress run"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/VadimNastoyashchy/cypress-aqa-cource.git"
+  },
+  "keywords": [],
```
Comment (VadimNastoyashchy) [inline package.json]: might me populated

**Review by VadimNastoyashchy — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (Rayman-bat9) [reply package-lock.json:1]: package-lock.json was added to gitignore file

File: package.json (hunk 2 above)
Comment (Rayman-bat9) [reply package.json]: keywords was added

File: package.json (hunk 1 above)
Comment (Rayman-bat9) [reply package.json]: author was added

**Review by marynaaa — APPROVED** [review]

**Review by VadimNastoyashchy — APPROVED** [review]

**Pull request merged**